scheduler/replica_scheduler.go

```diff
@@ -97,3 +125,17 @@ func (rcs *ReplicaScheduler) getNodeInfo() (map[string]*longhorn.Node, error) {
 	}
 	return scheduledNode, nil
 }
+
+func (rcs *ReplicaScheduler) generateReplica(replica *longhorn.Replica, diskCandidates map[string]*Disk) error {
```

Comment:
This function won't error out, so don't return `error`

And rename it to `scheduleReplicaToDisk`